### ISSUE TITLE
Scirpy dependency missing in requirements

### ISSUE THREAD

**Issue by felix0097** [theislab/sfaira#616]
**Describe the bug**
The `scirpy` dependency is missing in the requirements

**Comment by le-ander** [comment]
I guess we just add it @davidsebfischer ?

**Comment by davidsebfischer** [comment]
I was planning on having it as a optional dependency as it is only required to load particular data sets from raw files that have VDJ data. We could add list of optional dependencies that are required for data curation to document this better?

**Comment by le-ander** [comment]
ah ok. maybe a new entry in the extras_require in setup.py
so that people can call `pip install sfaira[all_datasets]` and get scirpy plus any other optional dependencies needed for dataloading

**Comment by davidsebfischer** [comment]
Yes, I would call this list `curation` though so that it is clear that this is independent of stores and database interfaces. I think that for now, scirpy is the only dependency there, in the future potentially muon and squidpy.
